resources/views/speakers/edit.blade.php

```diff
@@ -1,44 +1,94 @@
 @extends('layouts.app')
 
 @section('content')
-    <div>
-        <div>
-            <h1>Edit Speaker</h1>
-            @auth
-                <form method="post" action="{{ route('speakers.update', $speaker) }}" enctype="multipart/form-data">
-                    @method('PUT')
-                    @csrf
+    <div class="container md:w-[800px] mx-auto">
+        @auth
+            <form method="post" action="{{ route('speakers.update', $speaker) }}" enctype="multipart/form-data">
+                @method('PUT')
```

Comment:
Which method is correct? POST or PUT?

Reply:
This is a stupid Laravel feature: https://laravel.com/docs/9.x/blade#method-field

Reply:
Ok, got it.